content/en/platform/corda/5.2/application-networks/creating/members/virtual-node.md

```diff
@@ -55,6 +63,72 @@ $VIRTUAL_NODE_RESPONSE = Invoke-RestMethod -SkipCertificateCheck  -Headers @{Aut
 })
 ```
 
+Alternatively, if you are [using your own virtual node databases]({{< relref "../../../deploying-operating/vnodes/bring-your-own-db.md" >}}), specify the connection strings in the request body:
+
+```shell
+$X500_NAME = "C=GB, L=London, O=Alice"
+$VIRTUAL_NODE_RESPONSE = Invoke-RestMethod -SkipCertificateCheck  -Headers @{Authorization=("Basic {0}" -f $AUTH_INFO)} -Uri "$REST_API_URL/virtualnode" -Method Post -Body (ConvertTo-Json @{
+    request = @{
+        cpiFileChecksum = $CPI_STATUS_RESPONSE.cpiFileChecksum
+        x500Name = $X500_NAME
+        cryptoDdlConnection = @{
+            database = @{
+                jdbc = @{
+                    url = "jdbc:postgresql://localhost:5432/yourdatabase?currentSchema=vnode_crypto_FD545924FF37"
+                    user = "cryptoddlconnection_fd545924ff37"
+                    pass = "WXZS48pRyFQu0GNJNseRMeIL8ZczwfPAHnDJRZdQ11M6fJxjjdhbcKEYSwSiHHqT"
+                }
+            }
+        }
```

Comment:
These parameters will need to be a string instead of nested objects. The simplest change is to copy/paste the strings from the bash example:

```suggestion
        cryptoDdlConnection = "{\"database\":{\"jdbc\":{\"url\":\"jdbc:postgresql://localhost:5432/yourdatabase?currentSchema=vnode_crypto_FD545924FF37\"},\"user\":\"cryptoddlconnection_fd545924ff37\",\"pass\":\"WXZS48pRyFQu0GNJNseRMeIL8ZczwfPAHnDJRZdQ11M6fJxjjdhbcKEYSwSiHHqT\"}}"
```

I'm not sure if it could be fixed with a ConvertTo-Json call. I don't have a Windows computer to check. This may be possible but I'm not sure it will work:

```suggestion
        cryptoDdlConnection = ConvertTo-Json @{
            database = @{
                jdbc = @{
                    url = "jdbc:postgresql://localhost:5432/yourdatabase?currentSchema=vnode_crypto_FD545924FF37"
                    user = "cryptoddlconnection_fd545924ff37"
                    pass = "WXZS48pRyFQu0GNJNseRMeIL8ZczwfPAHnDJRZdQ11M6fJxjjdhbcKEYSwSiHHqT"
                }
            }
        }
```